configurations/s02_pipeline_configuration.py

```diff
@@ -41,11 +41,25 @@
                     FlowResultConfiguration("wusc_leap_s02e01_kalobeyei_activation", "rqa_s02e01", "leap_s02e01"),
                     FlowResultConfiguration("wusc_leap_s02e02_kalobeyei_activation", "rqa_s02e02", "leap_s02e02"),
                     FlowResultConfiguration("wusc_leap_s02e03_kalobeyei_activation", "rqa_s02e03", "leap_s02e03"),
+
+                    # Redirect s02e03_kalobeyei_follow_up to leap_s02e03
+                    FlowResultConfiguration("wusc_leap_s02e03_kalobeyei_follow_up", "probe_question", "leap_s02e03"),
```

Comment:
Is this definitely where you want to combine the datasets?

It's the simplest configuration but means it's harder to change your mind later. I think it depends on if you or RDA think there's any risk of you needing to split these out more than a short while into the future, so I can't tell which option is right here, just making sure you've thought about it.

Reply:
As explained above this is what we want to achieve.

Reply:
(For clarity, the other option would have been to combine in the coda/analysis configuration, which I didn't see discussed above)

Reply:
The team does not intend to keep this separate now and in the future. Combining in coda/analysis configuration would be great if that was the case. The follow-up questions were meant to enrich the qualitative analysis of the radio responses. They were sent to individuals who responded with _incomplete_ answers to the radio question. e.g response: we need support -> follow up: what kind of support? 

Hope this clarifies the need to merge the datasets.